plugins/opa/pkg/opa/opa.go

```diff
@@ -182,113 +155,6 @@ func getInsightsChecks() (clusterCheckModel, error) {
 	return jsonResponse, nil
 }
 
-func refreshLocalChecks(ctx context.Context, thisNamespace string) error {
-	client := kube.GetKubeClient()
-	logrus.Infof("Reconciling checks with Insights backend")
-
-	checkClient := client.DynamicInterface.Resource(checkGvr)
-	instanceClient := client.DynamicInterface.Resource(instanceGvr)
-
-	checkInstances, err := instanceClient.Namespace(thisNamespace).List(ctx, metav1.ListOptions{})
-	if err != nil {
-		return err
-	}
-	checks, err := checkClient.Namespace(thisNamespace).List(ctx, metav1.ListOptions{})
-	if err != nil {
-		return err
-	}
-	jsonResponse, err := getInsightsChecks()
-	if err != nil {
-		return err
-	}
-
-	logrus.Infof("Found %d checks in Insights, and %d checks in the cluster", len(jsonResponse.Checks), len(checks.Items))
-	logrus.Infof("Found %d check instances in Insights, and %d check instances in the cluster", len(jsonResponse.Instances), len(checkInstances.Items))
-
-	logrus.Infof("Deleting stale checks from the cluster")
-	for _, check := range checks.Items {
-		found := false
-		for _, supposedCheck := range jsonResponse.Checks {
-			if check.GetName() == supposedCheck.Name {
-				found = true
-				break
-			}
-		}
-		if !found {
-			err = checkClient.Namespace(check.GetNamespace()).Delete(ctx, check.GetName(), metav1.DeleteOptions{})
-			if err != nil {
-				return err
-			}
-
-		}
-	}
-
-	logrus.Infof("Deleting stale check instances from the cluster")
-	for _, instance := range checkInstances.Items {
-		found := false
-		for _, supposedInstance := range jsonResponse.Instances {
-			if instance.GetName() == supposedInstance.AdditionalData.Name {
-				found = true
-				break
-			}
-		}
-		if !found {
-			err = instanceClient.Namespace(instance.GetNamespace()).Delete(ctx, instance.GetName(), metav1.DeleteOptions{})
-			if err != nil {
-				return err
-			}
-		}
-	}
-
-	logrus.Infof("Updating checks in the cluster")
-	for _, supposedCheck := range jsonResponse.Checks {
-		found := false
-		for _, check := range checks.Items {
-			if check.GetName() == supposedCheck.Name {
-				found = true
-				break
-			}
-		}
-		newCheck := supposedCheck.GetUnstructuredObject(thisNamespace)
```

Comment:
@makoscafee I believe this method `GetUnstructuredObject` is no longer used and could be deleted? It's in `pkg/opa/types.go`.

Reply:
done